inspire_dojson/api.py

```diff
@@ -91,8 +90,6 @@ def marcxml2record(marcxml):
         return hepnames.do(marcjson)
     elif 'institution' in collections:
         return institutions.do(marcjson)
-    elif 'job' in collections or 'jobhidden' in collections:
-        return jobs.do(marcjson)
```

Comment:
can you `raise NotImplementedError("Jobs are not supported any more")` here instead? otherwise they will be treated as `hep`.

Reply:
:+1: